CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - Support `fetch_latest_metadata` option for crud requests with metadata (#335)
 - Support `noreturn` option for data change crud requests (#335)
 - Support `crud.schema` request (#336)
+- Support Tarantool tuples over IPROTO in crud (#337)
```

Comment:
go-tarantool does not depends on crud, the fix is related only to test infrastructure